lib/Lift/InstructionTranslator.cpp

```diff
@@ -681,7 +576,7 @@ IT::newInstruction(PTCInstruction *Instr,
     }
   }
 
-  Variables.newBasicBlock();
+  //Variables.newBasicBlock();
```

Comment:
This is a consequence of the lifetime change in QEMU. Smallest scope for variable lifetime pre-TCG optimization is now an extended basic block, which is a basic block that allows for early exits using conditional branches.

BB < EBB <TB

The allocated variables are instead reset on encountering an unconditional jump or exittb.

I left the comment around because I was expecting a bug related to variable lifetimes and I wanted to find the old reset point easily

Reply:
I wonder if it really makes sense to distinguish them in our context?
Under the assumption that the TCG we get is valid, I don't think we really care about the distinction.

Are they in different "namespaces"? As in, can you have `LIBTCG_TEMP_EBB` with ID 1 and a `LIBTCG_TEMP_TB` with the same ID?